### DISCUSSION TITLE
LIVE notification

### DISCUSSION THREAD

**Discussion by brixxnatt** [ant-media/Ant-Media-Server#6260] [General]
so I have 2 channels on AMS , and i have a laravel site , is  there a way to get notification on my site that this channel name is LIVE now
i mean what is the api that push notofication , so that i see how to make a script on my site to read that , please let me know
thanks

**Comment by Mohit-3196** [comment]
Hi @brixxnatt ,
How are you?

You can make use of the webhooks function on the Ant Media Server to get a webhook notification when the stream/channel is live. 
You can check how to set webhooks from here https://antmedia.io/docs/guides/advanced-usage/webhooks/

Let me know if it's fine.

Thank you,
Mohit
